pages/charcuterie.js

```diff
@@ -16,9 +21,15 @@ export default function Charcuterie() {
         <h2>Footer</h2>
         <Footer />
       </main>
+      <div style={{display: 'flex', alignItems: 'center', justifyContent: 'space-between', backgroundColor: 'rgb(44, 38, 66)', padding: '20px'}}>
```

Comment:
Are we going to have to create a wrapper for resource cards eventually? Maybe make and export an actual component for this

Reply:
we will but it probably will be pree different from this, haven't put too much thought into it yet, was gonna handle it in my future ticket :)